src/test/java/io/github/jamsesso/jsonlogic/FilterExpressionTests.java

```diff
@@ -15,12 +16,13 @@ public void testMap() throws JsonLogicException {
                   "  {\"var\": \"\"},\n" +
                   "  {\"==\": [{\"%\": [{\"var\": \"\"}, 2]}, 0]}\n" +
                   "]}";
-    int[] data = new int[] {1, 2, 3, 4, 5, 6};
-    Object result = jsonLogic.apply(json, data);
+    BigDecimal[] data = new BigDecimal[]{ BigDecimal.valueOf(1), BigDecimal.valueOf(2), BigDecimal.valueOf(3),
```

Comment:
What happens when we pass integer or double as input. Add tests for that.

Reply:
Everything is converted to BigDecimal while execution, adding more tests for that.